Makefile

```diff
@@ -324,6 +338,41 @@ openstack_deploy_cleanup: ## cleans up the service instance, Does not affect the
 	oc kustomize ${DEPLOY_DIR} | oc delete --ignore-not-found=true -f -
 	rm -Rf ${OPERATOR_BASE_DIR}/openstack-operator ${DEPLOY_DIR}
 
+.PHONY: edpm_deploy_prep
+edpm_deploy_prep: export KIND=OpenStackDataPlane
+edpm_deploy_prep: export EDPM_ANSIBLE_SECRET=${DATAPLANE_ANSIBLE_SECRET}
+edpm_deploy_prep: export EDPM_SINGLE_NODE=${DATAPLANE_SINGLE_NODE}
+edpm_deploy_prep: export EDPM_COMPUTE_IP=${DATAPLANE_COMPUTE_IP}
+edpm_deploy_prep: export EDPM_COMPUTE_1_IP=${DATAPLANE_COMPUTE_1_IP}
+edpm_deploy_prep: export OPENSTACK_RUNNER_IMG=${DATAPLANE_RUNNER_IMG}
+edpm_deploy_prep: export EDPM_NETWORK_CONFIG_TEMPLATE=${DATAPLANE_NETWORK_CONFIG_TEMPLATE}
+edpm_deploy_prep: export EDPM_SSHD_ALLOWED_RANGES=${DATAPLANE_SSHD_ALLOWED_RANGES}
+edpm_deploy_prep: export EDPM_CHRONY_NTP_SERVER=${DATAPLANE_CHRONY_NTP_SERVER}
+edpm_deploy_prep: export EDPM_OVN_METADATA_AGENT_NOVA_METADATA_HOST=${DATAPLANE_OVN_METADATA_AGENT_NOVA_METADATA_HOST}
+edpm_deploy_prep: export EDPM_OVN_METADATA_AGENT_PROXY_SHARED_SECRET=${DATAPLANE_OVN_METADATA_AGENT_PROXY_SHARED_SECRET}
+edpm_deploy_prep: export EDPM_OVN_METADATA_AGENT_BIND_HOST=${DATAPLANE_OVN_METADATA_AGENT_BIND_HOST}
+edpm_deploy_prep: export EDPM_OVN_METADATA_AGENT_TRANSPORT_URL=$(shell oc get secret rabbitmq-transport-url-neutron-neutron-transport -o json | jq -r .data.transport_url | base64 -d)
+edpm_deploy_prep: export EDPM_OVN_METADATA_AGENT_SB_CONNECTION=$(shell oc get ovndbcluster ovndbcluster-sb -o json | jq -r .status.dbAddress)
+edpm_deploy_prep: export EDPM_OVN_DBS=$(shell oc get ovndbcluster ovndbcluster-sb -o json | jq -r '.status.networkAttachments."openstack/internalapi"[0]')
+edpm_deploy_prep: $(if $(findstring true,$(DATAPLANE_NETWORK_ISOLATION)), nmstate nncp netattach metallb metallb_config) ## prepares the CR to install the data plane
```

Comment:
running `nmstate nncp netattach metallb metallb_config`  is required for the ctlplane if network isolation is used. should we make the parameter a generic `NETWORK_ISOLATION` and run this in `openstack_deploy_prep`, or just deploy the dependencies from now on per default in `openstack_deploy_prep`?

Reply:
We need two things actually for the control plane with network isolation:
1) make nmstate nncp netattach metallb metallb_config
2) config/samples/core_v1beta1_openstackcontrolplane_network_isolation.yaml make openstack_deploy

I think it would make sense to have NETWORK_ISOLATION a global flag to turn on network isolation both for the control and the edpm nodes. 
I have no problem doing #1) both as part of edpm_deploy_prep and as part of openstack_deploy_prep. So we have more flexibility on the order of make targets.
I think NETWORK_ISOLATION flag should instruct the openstack_deploy target to use the  config/samples/core_v1beta1_openstackcontrolplane_network_isolation.yaml sample by default.

For me this is OK to be done here or in a follow up. I can adapt to it any ways.

Reply:
done: https://github.com/fao89/install_yamls/blob/rmedpmplay/Makefile#L22-L29
@raukadah I've changed the var name to `NETWORK_ISOLATION`